CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - **Feature:** Added `getUniqueId` new interface for invoking data using `onSuccess` and `onError`.
 - **Feature:** Support `mailto` uri address. See [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a).
 - **Feature:** Added `rakuten.miniapp.user.action.SEND_MESSAGE` custom permission and applied to `ChatBridgeDispatcher.sendMessageToContactId`.
+- **Change:** Updated `getAccessToken` result with new Error type i.e `AccessTokenErrorType` to support more errors
```

Comment:
I think, if we still keep the old one, then we can replace the word "Updated" here by "Added" with mentioning that the old one is being deprecated.